package.json

```diff
@@ -20,10 +20,12 @@
     "next": "12.3.4",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "save-dev": "^0.0.1-security"
+    "save-dev": "^0.0.1-security",
```

Comment:
What's `save-dev` used for?

Reply:
Good catch, this was added accidentally, must have had a typo while using the --save-dev flag.